### ISSUE TITLE
3/27 +Microsoft Dynamics CRM - Phase I (email) + 8

### ISSUE THREAD

**Issue by anazariz** [lzim/teampsd#1167]
**1. Add description:** 

<<Microsoft Dynamics CRM - Phase I (email) >>
 
**2. Click on the right:**  
i. **Pipelines** - prioritize in a TeamPSD (& Workgroup) pipeline  
ii. **Assignee** - assign to an individual team member producing work for the issue  
iii. **Label**- assign workgroup (lead) responsible for tracking the task  
iv. **Projects** - assign to domain-specific TeamPSD tracker (issues, features, document, manuscript, team)  
v. **Milestones** - select monthly sprint milestone within the workgroup   
vi. **Estimate** - Estimate task in complexity points   
vii. **Epics** - assign to team-wide epic and TeamPSD master plan project  
viii. **Releases** - assign to key team release grouped with other relevant issues  

**3. Once you hit "Submit new issue**, underneath the first comment:   
viiii. **Add dependency** - assess for and add dependencies *blocking* & *blocked by* this issue

**Comment by anazariz** [comment]
@staceypark  i need access to the help and info mailboxes please.

**Comment by lzim** [comment]
> @staceypark i need access to the help and info mailboxes please.

@staceypark I’d like to add everyone at HQ to the two inboxes

**Comment by anazariz** [comment]
@lzim , @staceypark @anthonycpichardo 

Created an Excel workbook that reads inbound emails to MTL Help and MTL Info as well as outbound emails from these mailboxes. Please see my work area, email folder, emailer Excel workbook.  Please do not click the button yet.

**Comment by lijenn** [comment]
From HQ Workgroup 3/19:

We need @lzim @staceypark @branscombj @dlkibbe to review this as well.
Need to find existing meeting time ASAP before the QIICS review it.

**Comment by branscombj** [comment]
@lijenn @staceypark Can you give @dlkibbe and me some guidance on what to do here - I don't see anything to review. Thanks!

**Comment by dlkibbe** [comment]
@branscombj from the call yesterday, I believe you, @dlounsbu and I need to log in to the VA VPN and see if we can see the MTL Help and MTL Info inboxes in our VA email system.

**Comment by dlkibbe** [comment]
@lzim @staceypark @branscombj I logged into the VA VPN and was able to see MTL Help in my VA Outlook Inbox. I responded to a test message from @jamesmrollins to see if he would receive it. I thought I heard on the call yesterday that I should see an MTL Help option and an MTL Info option but only see an MTL Help inbox.

**Comment by branscombj** [comment]
> @lzim @staceypark @branscombj I logged into the VA VPN and was able to see MTL Help in my VA Outlook Inbox. I responded to a test message from @jamesmrollins to see if he would receive it. I thought I heard on the call yesterday that I should see an MTL Help option and an MTL Info option but only see an MTL Help inbox.

I am not able to use the VPN due (it looks like) to high traffic volume. Will try again periodically.

**Comment by staceypark** [comment]
@dlkibbe @branscombj 

Thanks Debbie for checking that. Both you and Jane, should only see MTL Help which is where the frontline teams we are working with for _MTL_ will be emailing. I've added all the facilitators and HQ to the MTL Help inbox and HQ to the MTL Info inbox (#1201).

The additional piece that Ash is working on is related to connecting that inbox with a CRM system so that specific teams and their emails can be mapped to the appropriate facilitators, etc. (You may remember Lindsey mentioning this previously during the Facilitate meetings a while ago). 

Jenn is working on figuring out a time in an existing meeting next week to slot the testing for the CRM piece from Ash. 
1. We plan to do an initial test and review first with Debbie, Jane, Lindsey, and Stacey. 
2. Then, we will edit and change anything based on the feedback.
3. Then, we will test and review with a few QIICs and get their feedback.

Anthony is writing up instructions for this test.

Hope this clears everything up for everyone.

FYI: @lijenn @anazariz @anthonycpichardo

**Comment by lijenn** [comment]
@dlkibbe @branscomb @staceypark @lzim We have decided to use the last half hour of next Monday, March 23rd's #MM meeting to discuss CRM requirements.

**Comment by anazariz** [comment]
@lzim , @anthonycpichardo , @staceypark , @lijenn 

Prior to gathering detailed use-case requirements from users, I worked on several inbound email distribution mechanisms. Having an understanding of a viable email redistribution mechanism is independent of how that mechanism will be used to provide a solution to end-users based on their requirements.

Mechanisms tested: 

1-Outlook - Event Driven (Not workable): 
Created an Outlook-based event-driven, which intercepted inbound emails to mtl.help and mtl.info and redirected them according to a user configurable list in Excel. This solution was deleted by the nightly system cleanup process.

2-Outlook – User Driven (Not workable):
Created a similar Outlook-based non-event-driven solution, requiring the user to click a button to invoke the process. This solution was also deleted by the nightly cleanup process.  

**3-Excel-Based (Workable – current solution):**
Created a similar Excel-based solution, which seems to be the only workable solution given the current circumstances.

4-Server-Based (May be Workable but will not be ready in time):
According to my experience, VA deletes Outlook customization on a virtual machine. However, it may allow it on a server (like an Exchange Server). This solution may not be viable from the perspective of the time that is required to implement it.    

5 - VA 365-Based (Workable - may be the best solution - I recommend it for Phase II):
Using my VA 365 account’s Power Automate (I do not have a pro account) I created a simple test for redirecting mtl.help and mtl.info to other emails. It worked fine. The problem is that I don’t have a data store (e.g. Excel or Access) with my VA account to hold the redirection rules. Without that all emails have to be hard-coded, which would make the solution unsustainable.

**Comment by lijenn** [comment]
From last half hour of #mm on 3/23:

Feedback Questions:
1. What's the process for looping in the technical team for questions? 
2. Do we need to forward to them for their response or grab an answer from the team and then respond on our end?

**Comment by lzim** [comment]
> From last half hour of #mm on 3/23:
> 
> Feedback Questions:
> 
> 1. What's the process for looping in the technical team for questions?
> 2. Do we need to forward to them for their response or grab an answer from the team and then respond on our end?

@lijenn As we discussed during our #hqhuddle this particular question should cross-ref the **Team PSD Manual** #1192

**Comment by anazariz** [comment]
@lzim , @anthonycpichardo 
If OK with you, I'l like to move this portion of the solution to done.  There is not much more I can do from the perspective of creating a prototype with our existing capabilities. Should we decide to purse a new solution (please see the PowerPoint I sent to you on 3/24 describing our option) I can start a new ticket.

**Comment by anthonycpichardo** [comment]
Closing this as we've addressed the current requirement given available resources. Issue #1235 exists to continue this in the future
